variants/variants.go

```diff
@@ -48,6 +48,8 @@ type Variant struct {
 	SoloSupplyCenters int
 	// SVG representing the variant map graphics.
 	SVGMap func() ([]byte, error) `json:"-"`
+	// A version for the map graphics (for use in caching mechanisms).
+	SVGMapVersion string
```

Comment:
Call it SVGVersion instead, and use it for caching of the unit resources as well.